src/main/java/org/team1540/base/power/PowerManager.java

```diff
@@ -122,23 +151,44 @@ private void stopScaling() {
   }
 
 
+  /**
+   * Determines if the current is currently spiking. If power limiting is not engaged,
+   * returns pdp.getTotalCurrent() &gt; currentSpikePeak.
+   * If power limiting is engaged, returns pdp.getTotalCurrent() &gt; currentTarget - currentMargin.
+   *
+   * @return Boolean representing if the current is spiking.
+   */
+  public boolean currentIsSpiking() {
+    if (!currentTimeHasPassed()) {
+      return pdp.getTotalCurrent() > currentSpikePeak;
+    } else {
+      return pdp.getTotalCurrent() > currentTarget - currentMargin;
+    }
+  }
+
   /**
    * Determines if the voltage is currently spiking. If power limiting is not engaged,
-   * returns pdp.getTotalCurrent() &gt; spikePeak. If power limiting is engaged, returns
-   * target - pdp.getTotalCurrent() &gt; margin.
+   * returns RobotController.getBatteryVoltage() &lt; voltageDipLow || RobotController.isBrownedOut();
+   * If power limiting is engaged, returns pdp.getVoltage() &lt; voltageDipLow + voltageMargin ||
+   * RobotController.isBrownedOut();.
    *
    * @return Boolean representing if the voltage is spiking.
    */
-  public boolean isSpiking() {
-    if (!timeHasPassed()) {
-      return pdp.getTotalCurrent() > spikePeak;
+  public boolean voltageIsDipping() {
```

Comment:
Refactor to `isVoltageDipping()` in accordance with [styleguide](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names).